rasa/nlu/selectors/response_selector.py

```diff
@@ -366,9 +387,13 @@ def process(self, message: Message, **kwargs: Any) -> None:
         )
 
         prediction_dict = {
-            "response": self._full_response(label) or {TEXT: label.get("name")},
+            "response": {
```

Comment:
As we changed this, do we need to update the components page in our docs?

Reply:
Yes, I'll update the docs in a separate PR.